pyanaconda/ui/gui/spokes/subscription.glade

```diff
@@ -415,6 +415,7 @@
                                         <property name="visible">True</property>
                                         <property name="can_focus">True</property>
                                         <property name="receives_default">False</property>
+                                        <property name="tooltip_text" translatable="yes">Red Hat Insights aims to increase your IT efficiency and speed across hybrid infrastructures by identifying and prioritizing risks, managing vulnerabilities, and compliance, and analyzing costs.  For more information, visit the Red Hat Insights information page.</property>
```

Comment:
I know it's copied from the bug, but two spaces between sentences don't look good:
> ...costs.&nbsp;&nbsp;For...

Reply:
Fixed.